test/test_nn.py

```diff
@@ -11888,7 +11888,19 @@ def test_cross_entropy_large_tensor(self, device, reduction):
         self.assertTrue(torch.allclose(loss.cpu(), loss_cpu, rtol=rtol, atol=atol))
         if reduction != "none":
             self.assertTrue(torch.allclose(logits.grad.cpu(), logits_cpu.grad, rtol=rtol, atol=atol))
-
+            
+    @parametrize_test("reduction", ("none", "mean", "sum"))
+    def test_cross_entropy_dim_parameter(self, device, reduction):
+        loss_1 = nn.CrossEntropyLoss(reduction=reduction)
+        loss_2 = nn.CrossEntropyLoss(reduction=reduction, dim=2)
+        input_1 = torch.randn(15, 10, 20, dtype=torch.float, device=device)
+        input_2 = input_1.swapaxes(2, 1)
+        target = torch.empty(15, 20, dtype=torch.long, device=device).random_(10)
+        if reduction != "none":
```

Comment:
could we add testing for the reduction = "none" case as well

Reply:
Tried adding this